scripts/dash-all-results-exploration.py

```diff
@@ -0,0 +1,589 @@
+        ),
+        html.Div(
+            [
+                dcc.Graph(
+                    id="crossfilter-indicator-scatter",
+                )
+            ],
+            style={
+                "display": "inline-block",
+                "padding": "0 20",
+                # "float": "left",
+            },
+        ),
+        html.Div(
+            [
+                dcc.Graph(
+                    id="crossfilter-indicator-contour",
+                )
+            ],
+            style={
+                "display": "inline-block",
+                "padding": "0 20",
+                # "float": "right",
+            },
+        ),
+        html.Div(
+            [
+                dcc.Graph(
+                    id="by-compound",
+                )
+            ],
+            style={
+                "display": "inline-block",
+                "padding": "0 20",
+                # "float": "right",
+            },
+        ),
+        html.Div(
+            [
+                dcc.Graph(
+                    id="by-structure",
+                )
+            ],
+            style={
+                "display": "inline-block",
+                "padding": "0 20",
+                # "float": "right",
+            },
+        ),
+        html.Div(
+            [
+                html.H4("By Structure (Counts)"),
+                dcc.Graph(id="per-structure-bar-chart"),
+            ],
+            style={"width": "49%", "display": "inline-block"},
+        ),
+        html.Div(
+            [
+                html.H4("By Structure (Values)"),
+                dcc.Graph(id="per-structure-bar-chart2"),
+            ],
+            style={"width": "49%", "display": "inline-block"},
+        ),
+        dash_table.DataTable(
+            df.to_dict("records"),
+            id="table",
+            columns=[
+                {"name": column, "id": column}
+                for column in [
+                    "Compound_ID",
+                    "Structure_Source",
+                    "RMSD",
+                    "POSIT",
+                    "Chemgauss4",
+                ]
+            ],
+            style_table={
+                "width": "50%",
+                "float": "center",
+                "display": "inline-block",
+                "padding": "0 20",
+                # "float": "right",
+            },
+            filter_action="custom",
+            filter_query="",
+        ),
+        html.Div(
+            className="row",
+            children=[
+                html.Div(
+                    [
+                        dcc.Markdown(
+                            """
+                            **Click Data**
+    
+                            Click on points in the graph.
+                        """
+                        ),
+                        html.Pre(id="click-data", style=styles["pre"]),
+                    ],
+                    className="three columns",
+                ),
+            ],
+        ),
+    ]
+)
```

Comment:
We probably want to have this in a separate file that we can just import, such that this script is easier to read and handle, when needed.

Reply:
I agree! this is beyond me right now though